### PR TITLE
make work

### PR DESCRIPTION
**Pull request by LunarTwilight**
not sure if the header is needed, and the instance var was taken from main.js

### CODE REVIEW

**Comment by LunarTwilight** [comment]
think the syntax error might be a false positive, the JS I edited for the PR looks about the same as the JS I edited on where I have it running right now.

File: main.js [hunk 1]
```diff
@@ -10,10 +10,10 @@
  */
 import {CookieJar} from 'tough-cookie';
 import {WebhookClient} from 'discord.js';
-import config from './config.json';
+import config from './config.json' assert { type: 'json' };
```
Comment (KockaAdmiralac) [inline main.js:13]: Let's not use these until they become a stable ECMAScript feature, supported by ESLint. Importing JSON modules without assertions works in Node v16, so you can change the `node` engine in `package.json` to `16.x`.

File: main.js [hunk 2]
```diff
@@ -92,13 +92,14 @@ class ContentReviewLog {
...
-            await this._http.post(`https://services.${config.domain}/auth/token`, {
+            await this._http.post(`https://services.${config.domain}/mobile-fandom-app/fandom-auth/login`, {
                 form: {
                     password: config.password,
                     username: config.username
                 },
                 headers: {
-                    'X-Wikia-WikiaAppsID': 1234
+                    'X-Wikia-WikiaAppsID': 1234,
+                    'X-Fandom-Auth': 1
```
Comment (KockaAdmiralac) [inline main.js:102]: ESLint complains about keys not being sorted in ascending order here.

**Review by KockaAdmiralac — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (LunarTwilight) [reply main.js:13]: It didn't work for me on node 17.9.0 without them

Comment (KockaAdmiralac) [reply main.js:13]: Yeah, but Node v16 is the current LTS.

Comment (LunarTwilight) [reply main.js:13]: Hm it looks like it already is set on package.json?
```json
    "engines": {
        "node": ">=16"
    },
```

Comment (KockaAdmiralac) [reply main.js:13]: It is set to `>=16`, implying that Node v17 would work. Since that is inaccurate, it should be changed to `16.x`.

Comment (LunarTwilight) [reply main.js:13]: alright

Comment (KockaAdmiralac) [reply main.js:13]: (You also have to not actually use import assertions to fix LGTM and ESLint errors.)

Comment (LunarTwilight) [reply main.js]: Alright removed, hopefully it still works b/c it didn't work for me without them.

**Review by KockaAdmiralac — APPROVED** [review]
Thanks!

**Pull request merged**